core/modules/widgets/checkbox.js

```diff
@@ -3,167 +3,118 @@ title: $:/core/modules/widgets/checkbox.js
 type: application/javascript
 module-type: widget
 
-Checkbox widget
+The Checkbox widget toggles the value of a field between two string values
 
 \*/
-(function(){
+(function () {
 
-/*jslint node: true, browser: true */
-/*global $tw: false */
-"use strict";
+	/*jslint node: true, browser: true */
+	/*global $tw: false */
```

Comment:
IMO code formatting should not be done by hand. There are code formatters out there. We just need to discuss and agree on the defaults. see: http://jscs.info/ .. There are some default settings, that would work very well with TW code. .. we just neet to agree on something. .. which can be a minefield :)